packages/patternfly-4/react-core/src/components/Nav/NavGroup.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styles from '@patternfly/patternfly-next/components/Nav/styles.css';
+import a11yStyles from '@patternfly/patternfly-next/utilities/Accessibility/styles.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import NavToggle from './NavToggle';
+import { AngleRightIcon } from '@patternfly/react-icons';
+
+const propTypes = {
+  id: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  ariaLabel: PropTypes.string.isRequired,
+  expandable: PropTypes.bool.isRequired,
+  expanded: PropTypes.bool.isRequired,
+  children: PropTypes.node.isRequired,
+  innerClassName: PropTypes.string.isRequired,
+  groupId: PropTypes.number.isRequired,
+  active: PropTypes.bool.isRequired
```

Comment:
rename props for consistency
expandable -> isExpandable
active -> isActive
exapanded -> isExpanded